blocks/api/raw-handling/index.js

```diff
@@ -89,9 +89,12 @@ export default function rawHandler( { HTML = '', plainText = '', mode = 'AUTO',
 	}
 
 	// Normalize unicode to use composed characters.
+	// Unsupported in IE.
```

Comment:
Should we clarify why it's okay that we don't normalize in IE? I'm not entirely clear about it myself.

Reply:
Hopefully #7105 does the trick.